bin/index.js

```diff
@@ -8,8 +8,20 @@ const runReactAppBlueprint = require('../blueprints/react-app');
 
 const usage = chalk.cyan(`\nHi, I'm Gavin! 👋\nUsage: gavin [command] [options]`);
 
+// TODO: Would be nice to have a bundler handle syncing the volta version with this one
+const nodeMajor = 18;
```

Comment:
```suggestion
const pkg = require('../package.json');
const nodeMajor = Number(pkg.volta.node.split('.')[0]);
```

This should work, but definitely test it out

Reply:
![Screenshot 2023-10-17 at 11 00 54 AM](https://github.com/Gavant/cli/assets/86789397/21efaa6d-fc27-46b7-a4d0-6ae5ca3df176)

:nice: